ui/app/components/auth-form.js

```diff
@@ -237,6 +241,15 @@ export default Component.extend(DEFAULTS, {
     }
   }).withTestWaiter(),
 
+  delayAuthMessageReminder: task(function*() {
+    if (Ember.testing) {
+      this.showLoading = true;
```

Comment:
Why do we showLoading in testing scenario but not otherwise?

Reply:
Because conditionally, to show the message we have to have both showLoading === true &&& this task function no longer running.